test/functional/test_site/expected/bugs/index.html

```diff
@@ -55,7 +55,7 @@
   </div>
 </body>
 <script src="../markbind/js/vue.min.js"></script>
-<script src="../markbind/js/vue-strap.min.js"></script>
+<script src="../markbind/js/components.min.js"></script>
```

Comment:
the only test file changes are from this and renaming `vue-strap.min.js` to `component.min.js`